clang/test/CXX/drs/cwg27xx.cpp

```diff
@@ -201,3 +201,31 @@ static_assert(false, f().s);
 #endif
 } // namespace cwg2798
 
+namespace cwg2707 { // cwg2707: 20
```

Comment:
Given the number of the Core issue, this test should've been placed towards the beginning of the file. We keep them in the ascending order.